app/views/recipes/index.html.erb

```diff
@@ -1,11 +1,19 @@
 <h2>Results</h2>
-
-<% @recipes.each do |recipe| %>
-  <div class="recipes">
-    <%= link_to recipe_path(recipe.id) do %>
-      <%= recipe.title %><br>
-      <%= image_tag recipe.image %><br>
-    <% end %>
-    <p>You're missing <%= recipe.ingredients.missing_ingredients %> Ingredients.</p>
+<div class="container">
+  <div class="row">
+  <% @recipes.each do |recipe| %>
+    <div class="col-sm-4">
+      <div class="recipes">
+        <div class="thumbnail">
+          <%= image_tag recipe.image %>
+          <h4><%= recipe.title %></h4>
+          <div class="caption">
+          <p>You will need <%= recipe.ingredients.missing_ingredients %> more ingredients to make this dish.</p>
```

Comment:
Does this message fit 3 times side by side on the recipes index page? What does it look like if the screen is being resized.